src/AddAggregatesPlugin.ts

```diff
@@ -239,6 +288,80 @@ const AddAggregatesPlugin: Plugin = builder => {
         },
         {}
       ),
+      ...pgIntrospectionResultsByKind.procedure.reduce(
+        (memo: GraphQLFieldConfigMap<any, any>, proc: PgProc) => {
+          /* TODO: Do fields need to be omitted? */
+          const attrIsNumberLike =
+            pgIntrospectionResultsByKind.typeById[proc.returnTypeId]
+              .category === "N";
+          if (attrIsNumberLike && pgColumnFilter(proc, build, context)) {
+            const computedColumnDetails = getComputedColumnDetails(
+              build,
+              table,
+              proc
+            );
+            if (computedColumnDetails) {
+              const fieldName = inflection.computedColumn(
+                computedColumnDetails.pseudoColumnName,
+                proc,
+                table
+              );
+              return build.extend(memo, {
+                [fieldName]: pgField(
+                  build,
+                  fieldWithHooks,
+                  fieldName,
+                  ({ addDataGenerator }: any) => {
+                    addDataGenerator((parsedResolveInfoFragment: any) => {
+                      return {
+                        pgQuery: (queryBuilder: QueryBuilder) => {
+                          // Note this expression is just an sql fragment, so you
+                          // could add CASE statements, function calls, or whatever
+                          // you need here
+                          const expr = sql.fragment`${sql.identifier(
+                            proc.namespaceName,
+                            proc.name
+                          )}(${queryBuilder.getTableAlias()})`;
+                          queryBuilder.select(
+                            // You can put any aggregate expression here; I've wrapped it in `coalesce` so that it cannot be null
+                            sql.fragment`coalesce(sum(${expr}), 0)`,
+                            // We need a unique alias that we can later reference in the resolver
+                            getSafeAliasFromAlias(
+                              parsedResolveInfoFragment.alias
+                            )
+                          );
+                        },
+                      };
+                    });
+                    return {
+                      description: `Sum of ${fieldName} across the matching connection`,
+                      type: new GraphQLNonNull(GraphQLFloat), // TODO: not necessarily the correct type
```

Comment:
I think this is the safest type for now - `sum(int)` could end up being too large for `int`; float gives us a huge range (but isn't super precise). If necessary we could switch to BigInt / BigFloat later.

Reply:
If we're doing `sum(decimal)` we probably ought to expose a BigFloat..